### ISSUE TITLE
Some reserved words can not be used in strict mode

### ISSUE THREAD

**Issue by ariya** [eslint/espree#198]
Discovered through eslint-ing some code. The simplest test case follows:

```js
espree.parse('"use strict"; static = 42')
```

Actual: a syntax tree.

Expected: a syntax error.

**Comment by nzakas** [comment]
Fixed in 3.0.0.
